ReSharper.FSharp/src/FSharp.ExternalFormatter.Host/FSharp.ExternalFormatter.Host.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net461</TargetFramework>
+        <AssemblyName>JetBrains.ReSharper.Plugins.FSharp.ExternalFormatter.Host</AssemblyName>
+        <OutputType>Exe</OutputType>
+        <SignAssembly>false</SignAssembly>
```

Comment:
Why this line is needed?

Reply:
in this pull request, we may not need it, but we will need it to load the custom Fantomas version.

Reply:
In what cases this line could be needed? Isn't it the default value?

Reply:
yes you seem to be right, it is false by default